### DISCUSSION TITLE
Fly.io 初体验之博客搬家

### DISCUSSION THREAD

**Discussion by giscus** [JmPotato/Pomash#11] [Announcements]
# Fly.io 初体验之博客搬家

https://ipotato.me/article/74

**Comment by ttys3** [comment]
博客我用的 Hugo.
最近刚好也在玩 fly.io. 很香.
反正免费额度比 azure app service 爽多了.
azure app service 是真难用...

**Comment by ttys3** [comment]
过几天有空了我也写一篇关于 fly.io 的

**Reply by puzzle9** [reply]
啊 插眼
